test_rfc6266_parser.py

```diff
@@ -56,42 +45,25 @@ def test_location_fallback():
     ).filename_unsafe == u'étoilé'
 
 
-def test_strict():
-    # Trailing ; means the header is rejected
-    assert parse_headers('attachment;').disposition == 'inline'
-    assert parse_headers('attachment; key=val;').disposition == 'inline'
-    try:
-        cd = parse_headers(
-            'attachment; filename="spa  ced";')
-    except ValueError:
-        assert True
-    else:
-        assert False, cd
-
-
 def test_relaxed():
-    assert parse_headers(
-        'attachment;', relaxed=True).disposition == 'attachment'
-    assert parse_headers(
-        'attachment; key=val;', relaxed=True).disposition == 'attachment'
-    cd = parse_headers(
-        'attachment; filename="spa  ced";',
-        relaxed=True)
+    assert parse_headers('attachment;').disposition == 'attachment'
+    assert parse_headers('attachment; key=val;').disposition == 'attachment'
+    cd = parse_headers('attachment; filename="spa  ced";')
     assert cd.filename_unsafe == u'spa ced'
 
 
-
-def test_roundtrip():
+@pytest.mark.parametrize("name", [
+    'a b', 'a b ', ' a b', 'a\"b', u'aéio o♥u"qfsdf!'
```

Comment:
no `'a   b'`?

Reply:
It was use case related to strict mode (`relaxed=False`).